python/tchannel/messages/call_response.py

```diff
@@ -27,3 +35,95 @@ class CallResponseMessage(CallRequestMessage):
         'arg_2',
         'arg_3',
     )
+
+    FLAGS_SIZE = 1
+    CODE_SIZE = 1
+    # sizeOf(span_id) + sizeOf(parent_id) + sizeOf(trace_id) = 3*8 = 24
+    TRACE_SIZE = 8
+    TRACEFLAGS_SIZE = 1
+    NH_SIZE = 1
+    HEADER_SIZE = 1
+    CSUMTYPE_SIZE = 1
+    CSUM_SIZE = CallRequestMessage.CHECKSUM[CSUMTYPE_SIZE]
+
+    ARG_LENGTH = 2
+
+    def parse(self, payload, size):
+        """Parse a call request message from a payload."""
+        self.flags = read_number(payload, self.FLAGS_SIZE)
+        self.code = read_number(payload, self.CODE_SIZE)
+
+        self.span_id = read_number(payload, self.TRACE_SIZE)
+        self.parent_id = read_number(payload, self.TRACE_SIZE)
+        self.trace_id = read_number(payload, self.TRACE_SIZE)
+
+        self.traceflags = read_number(payload, self.TRACEFLAGS_SIZE)
```

Comment:
this is just the method in the subclass overrides the base class method. There are some difference between subclass one and base class one.